component-library/src/app/pages/banner-documentation/banner-documentation.component.scss

```diff
@@ -31,3 +26,14 @@
     }
   }
 }
+
+::ng-deep {
```

Comment:
We should avoid using ::ng-deep directly. Can we add a component-scoped CSS class beforehand? 

Example:
.block ::ng-deep 

It's very low risk but it's still best practice to always include a parent selector before ::ng-deep